### DISCUSSION TITLE
Sensor attribute start time

### DISCUSSION THREAD

**Discussion by thatsamwilliams** [BottlecapDave/HomeAssistant-OctopusEnergy#260] [Q&A] [answered]
I’ve set a sensor up to give me the best 2.5 hr window of the day.

I see that the sensor will start at 12:30 (from attributes). How can I just display that time on a card?

thank you

**Comment by BottlecapDave** [comment, marked as answer]
You'll want to take advantage of [template sensors](https://community.home-assistant.io/t/template-return-state-as-datetime/325078) and use them to extract the [state_attr](https://www.home-assistant.io/docs/configuration/templating/) functionality to extract the value from the attributes of the sensor. You should then be able to use this sensor in an entity card on the dashboard.
